src/test/java/lotto/domain/NumberTest.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class NumberTest {
+
+    @Test
+    @DisplayName("로또 번호 정상 검증")
+    public void numberTest() {
+        assertThat(new Number(1)).isEqualTo(new Number(1));
+    }
```

Comment:
```suggestion
    @Test
    @DisplayName("로또 번호 정상 검증")
    public void numberTest() {
        // given
        Number target = new Number(1);

        // when
        Number expected = new Number(1);

        // then
        assertThat(target).isEqualTo(expected);
    }
```

피드백드린 given when then 패턴 적용이 안되고 있는 것 같아요 😢 
적용하시는데 어려움이 있으시면 코멘트나 DM으로 언제든 문의주세요 :)